packages/react/src/stories/ic-button.stories.mdx

```diff
@@ -1079,3 +1079,22 @@ export const PopoverDropdown = () => {
     )}
   </Story>
 </Canvas>
+
+### File Upload
+
+<Canvas>
+  <Story name="File Upload">
+    <IcButton
+      variant="primary"
+      file-upload="true"
+      accept=".doc, text/plain, .json"
+      onClick={(event) =>
+        document.addEventListener("icFileSelection", (event) =>
```

Comment:
I couldn't see anything getting logged to the console after i selected a file

Reply:
i wonder if the listener needs to be add onto the ic-button element rather than the document

Reply:
I have added an alert so it should be being displayed.